src/main/java/com/cburch/logisim/gui/appear/ClipboardContents.java

```diff
@@ -14,29 +14,18 @@
 import com.cburch.logisim.data.Location;
 import java.util.Collection;
 import java.util.Collections;
+import lombok.Getter;
 
 class ClipboardContents {
   static final ClipboardContents EMPTY = new ClipboardContents(Collections.emptySet(), null, null);
 
-  private final Collection<CanvasObject> onClipboard;
-  private final Location anchorLocation;
-  private final Direction anchorFacing;
+  @Getter private final Collection<CanvasObject> elements;
```

Comment:
This new name seems to be more generic and does not hint what is to be the content of the collection.